enterprise_access/apps/subsidy_request/models.py

```diff
@@ -379,6 +379,184 @@ def approve(self, reviewer):
         self.reviewed_at = localized_utcnow()
         self.save()
 
+    @classmethod
+    def annotate_dynamic_fields_onto_queryset(cls, queryset):
+        """
+        Annotate extra dynamic fields used by this viewset for DRF-supported ordering and filtering.
+
+        Fields added:
+        * latest_action_type (CharField) - Most recent action type
+        * latest_action_time (DateTimeField) - Most recent action timestamp
+        * latest_action_status (CharField) - Most recent action status
+        * latest_action_error_reason (CharField) - Most recent action error
+        * request_lifecycle_state (CharField) - Derived request lifecycle state
+        * lifecycle_state_sort_order (IntegerField) - Sort order for lifecycle states
+
+        Notes:
+        * This method uses the same complex ORM annotation pattern as content assignments.
+
+        Args:
+            queryset (QuerySet): LearnerCreditRequest queryset, vanilla.
+
+        Returns:
+            QuerySet: LearnerCreditRequest queryset, same objects but with extra fields annotated.
+        """
+        from datetime import datetime
```

Comment:
I think this is not the right way to do imports. We should define them at the top.